utils/__tests__/redis_storage.test.ts

```diff
@@ -88,66 +84,64 @@ describe("getTask", () => {
           value => expect(value).toEqual(aRedisValue)
         )
       )
-    )();
+    );
   });
 
-  it("should return none if no value was found for the provided key", async () => {
-    getMock.mockImplementationOnce((_, cb) => cb(undefined, null));
-    await pipe(
+  it("should return none if no value was found for the provided key", () => {
+    getMock.mockImplementationOnce(_ => undefined);
+    pipe(
       getTask(redisClientMock as any, aRedisKey),
       TE.bimap(
         () => fail(),
         maybeResult => expect(O.isNone(maybeResult)).toBeTruthy()
       )
-    )();
+    );
   });
 
-  it("should return an error if redis get value fails", async () => {
-    getMock.mockImplementationOnce((_, cb) =>
-      cb(new Error("Cannot get value"), null)
-    );
-    await pipe(
+  it("should return an error if redis get value fails", () => {
+    getMock.mockImplementationOnce(_ => new Error("Cannot get value"));
+    pipe(
       getTask(redisClientMock as any, aRedisKey),
       TE.bimap(
         _ => expect(_).toBeDefined(),
         () => fail()
       )
-    )();
+    );
   });
 });
 
 describe("existsTask", () => {
-  it("should return true if key exists in redis", async () => {
-    await pipe(
+  it("should return true if key exists in redis", () => {
+    pipe(
       existsKeyTask(redisClientMock as any, aRedisKey),
       TE.bimap(
         () => fail(),
         exists => expect(exists).toBeTruthy()
       )
-    )();
+    );
   });
 
-  it("should return false if key does not exists in redis", async () => {
-    existsMock.mockImplementationOnce((_, cb) => cb(null, 0));
-    await pipe(
+  it("should return false if key does not exists in redis", () => {
+    existsMock.mockImplementationOnce(_ => 0);
+    pipe(
       existsKeyTask(redisClientMock as any, aRedisKey),
       TE.bimap(
         () => fail(),
         exists => expect(exists).toBeFalsy()
       )
-    )();
+    );
   });
 
-  it("should return an error if redis exists fails", async () => {
-    existsMock.mockImplementationOnce((_, cb) =>
-      cb(new Error("Cannot recognize exists on redis"), null)
+  it("should return an error if redis exists fails", () => {
+    existsMock.mockImplementationOnce(
+      _ => new Error("Cannot recognize exists on redis")
```

Comment:
this should return a promise because we are using the promise API for the client
```suggestion
      _ => Promise.reject(new Error("Cannot recognize exists on redis"))
```

Reply:
fixed in https://github.com/pagopa/io-functions-cgn/pull/102/commits/1e5923a1d4661210ca8ea4767ed4016fd7ed2481